src/store.config.js

```diff
@@ -1,72 +1,18 @@
-import { createStore } from 'redux';
+import * as asyncInitialState from 'redux-async-initial-state';
+import axios from 'axios';
```

Comment:
both of those lines here not usre you need them anymore

Reply:
how did I miss this :disappointed: